convert2rhel/special_cases.py

```diff
@@ -0,0 +1,29 @@
+import logging
```

Comment:
Wondering if the file `special_cases.py` should be called something else, or just moved into pkghandler for now. The file name is rather undescriptive without context

Reply:
me either. Already raised this question one comment above

Reply:
ok, we are handling this in the thread above. closing this one